packages/Discretization/test/tstInterpolation.cpp

```diff
@@ -273,7 +273,9 @@ TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( Interpolation, one_topo_one_fe_three_dim,
     DataTransferKit::Mesh<DeviceType> mesh( cell_topologies, cells,
                                             coordinates );
     DataTransferKit::Interpolation<DeviceType> interpolation(
-        comm, mesh, points_coord, cell_dofs_ids, DTK_HGRAD );
+        *( Teuchos::rcp_dynamic_cast<Teuchos::MpiComm<int> const>( comm )
+               ->getRawMpiComm() ),
```

Comment:
There is no reason to build a Teuchos communicator. Just create a raw mpi communicator